features/node-upload-multiple.feature

```diff
@@ -0,0 +1,253 @@
+Feature: Node source map upload multiple
```

Comment:
In both these and the single node upload scenarios, we need to make sure the `minifiedUrl` parameter is correct.

Despite having a misleading name, it is still really important, because it is part of what is used to match an error's stackframe to the uploaded source map.

So in each scenario we need to assert that the provided `minifiedUrl` is the relative path from `projectRoot` to the bundle. For example, in the webpack example, it should be `dist/<bundle_name>.js`.

Reply:
👍 I think because there's no `--bundle-url` parameter I assumed there was no `minifiedUrl` either — I'll add it back to the tests

Reply:
Added checks for `minifiedUrl` in e6b523b